emily_cron/app/clients/mempool.py

```diff
@@ -7,11 +7,48 @@
 logger = logging.getLogger(__name__)
 
 
+def _collect_rbf_txids(data: dict[str, Any] | None) -> list[str]:
+    """Recursively collect all RBF transaction IDs from the replacement chain.
+
+    Args:
+        data: Transaction replacement data from mempool API
+
+    Returns:
+        List of transaction IDs that replaced the original transaction
+    """
+    txids = []
+
+    if data is None:
+        return txids
+
+    if tx := data.get("tx", {}):
```

Comment:
nit: since we are checking for falsy we don't need the default. Or we could keep it and just remove the `if` (since it would be caught by the `if txid := tx.get("txid")` anyway). Or even chain the two in a single if.
```suggestion
    if tx := data.get("tx"):
```